cmd/tsbs_load_mongo/creator.go

```diff
@@ -50,47 +50,69 @@ func (d *dbCreator) RemoveOldDB(dbName string) error {
 }
 
 func (d *dbCreator) CreateDB(dbName string) error {
-	cmd := make(bson.D, 0, 4)
-	cmd = append(cmd, bson.E{"create", collectionName})
+	createTSCmd := make(bson.D, 0, 4)
```

Comment:
Why rename this to `createTSCmd`? Seems like the new name would be inaccurate if `timeseriesCollection` is false. Maybe just `createCmd`?

Reply:
sure will rename it createCollCmd to be consistent with the shardCollCmd later